Adyen/ApiSerialization/Converter/SaleToPoiMessageSecuredConverter.cs

```diff
@@ -14,8 +14,6 @@ internal class SaleToPoiMessageSecuredConverter : JsonConverter
 
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            serializer.NullValueHandling = NullValueHandling.Ignore;
```

Comment:
See comment above, let's introduce a different function instead so that existing integrations don't break.

Reply:
The same as for `SaleToPoiMessageConverter`.